docs/operational_guide/kernel_tuning.md

```diff
@@ -0,0 +1,40 @@
+Kernel Tuning
+=============
+
+This document lists the Kernel tweaks `m3dbnode` needs to run well.
```

Comment:
is that what we actually call the binary? I always refer to it as `m3db` or `M3DB` when I'm writing documentation. We should probably be consistent about that

Reply:
yeah i'm onboard with being consistent. I'll change to `M3DB` for now.

Reply:
done